articles/managing_teams_for_groups_and_projects.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Managing Teams for Groups and Projects"
+layout: article
+excerpt: Using Synapse teams for group permissions in projects. 
+category: inpractice
+---
+
+# Creating Permissions Models for Synapse Projects
+
+Synapse supports several different kinds of project permissions. These are described in more detail as "sharing settings" in the article on [access controls](/articles/access_controls.html). Additionally, Synapse supports setting "local sharing settings" that govern permissions within a project; for example, if you want to make your project public while keeping some folders or data private, you would use local sharing settings to do this. Permissions and sharing settings within Synapse are otherwise hierarchical; that is, if you set permissions on a project, everything within that project inherits those permissions until local sharing settings have been created. 
```

Comment:
```suggestion
Synapse supports several different kinds of project permissions. These are described in more detail as "sharing settings" in the article on [access controls](/articles/access_controls.html#sharing-settings). Additionally, Synapse supports setting "local sharing settings" that allow you to make your project public while keeping some folders, files, or tables private. To do this, you would use local sharing settings. Permissions and sharing settings within Synapse are otherwise hierarchical; that is, if you set permissions on a project, everything within that project inherits those permissions until local sharing settings have been created. 
```